docs/data-sources/network.md

```diff
@@ -43,8 +43,6 @@ data "stackit_network" "example" {
 - `labels` (Map of String) Labels are key-value string pairs which can be attached to a resource container
 - `name` (String) The name of the network.
 - `nameservers` (List of String, Deprecated) The nameservers of the network. This field is deprecated and will be removed soon, use `ipv4_nameservers` to configure the nameservers for IPv4.
-- `no_ipv4_gateway` (Boolean) If set to `true`, the network doesn't have an IPv4 gateway.
```

Comment:
why do we remove these?

Reply:
These fields are not needed in the datasource, as they are not actually used nor returned by the API. They are just used to match the API behaviour on the gateway field:
- `null` --> no gateway
- a value --> this is used for the gateway
- field not present --> default gateway